test/js/api/shipment.js

```diff
@@ -0,0 +1,80 @@
+const assert = require("assert");
+const config = require("../config");
+const ripe = require("../../../src/js");
+
+describe("ShipmentAPI", function() {

```

Comment:
also add a test for delete shipment function [inspiration](https://github.com/ripe-tech/ripe-sdk/blob/9fd7461be6bef3b4e479d4a0df8e4b38c3bd6bf6/test/js/api/sku.js#L316)